src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -730,7 +730,7 @@ void CoilCoolingDX::simulate(EnergyPlus::EnergyPlusData &state,
         if (speedNum > 0) {
             // calculate and report condensation rates  (how much water extracted from the air stream)
             // water flow of water in m3/s for water system interactions
-            Real64 averageTemp = (evapInletNode.Temp - evapOutletNode.Temp) / 2.0;
+            Real64 averageTemp = (evapInletNode.Temp + evapOutletNode.Temp) / 2.0;
```

Comment:
corrected average water temperature calculation for evaluating water density.

Reply:
👍